scene/2d/tile_map.cpp

```diff
@@ -1325,6 +1346,7 @@ void TileMap::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("clear"), &TileMap::clear);
 
 	ObjectTypeDB::bind_method(_MD("get_used_cells"), &TileMap::get_used_cells);
+	ObjectTypeDB::bind_method(_MD("get_used_cells_by_id"), &TileMap::get_used_cells_by_id);
```

Comment:
You should bind the argument too, see e.g. `ObjectTypeDB::bind_method(_MD("world_to_map", "worldpos"), &TileMap::world_to_map);`